api/apiFunctions.ts

```diff
@@ -129,29 +165,34 @@ export const getCurrentWeek = async (): Promise<IGameWeek> => {
 /**
  * Get all weekly picks
  * @param props - The game ID and week ID
- * @param props.gameId - The game ID
+ * @param props.leagueId - The league ID
  * @param props.weekId - The week ID
  * @returns {IWeeklyPicks['userResults'] | null} - The user results or null
  */
 export async function getAllWeeklyPicks({
-  gameId,
+  leagueId,
   weekId,
 }: {
-  gameId: IGameGroup['currentGameId'];
+  leagueId: ILeague['leagueId'];
   weekId: IGameWeek['id'];
 }): Promise<IWeeklyPicks['userResults'] | null> {
   try {
     const response = await databases.listDocuments(
       appwriteConfig.databaseId,
       Collection.GAME_RESULTS,
-      [Query.equal('gameId', gameId), Query.equal('gameWeekId', weekId)],
+      [Query.equal('gameId', leagueId), Query.equal('gameWeekId', weekId)],
     );
 
     // check if any users have selected their pick
     if (response.documents[0].userResults === '') {
       return null;
     }
 
+    // check if any users have selected their pick
+    if (response.documents[0].userResults === '') {
```

Comment:
how come there are 2 of these doing the same thing? line 187 has an if statement doing the exact same as this if statement on line 192